packages/mco/models/disaster-recovery.ts

```diff
@@ -0,0 +1,15 @@
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk-internal-kubevirt/lib/api/common-types';
+
+
```

Comment:
```suggestion
import { K8sModel } from '@openshift-console/dynamic-plugin-sdk/lib/api/common-types';

```
Don't use kubevirt anywhere...